src/components/seller/exchanges/SellerExchanges.tsx

```diff
@@ -252,6 +252,7 @@ export default function SellerExchanges({
     async (exchange: Exchange, from?: Dayjs) => {
       try {
         const threadId: ThreadId = {
+          // TODO: add chainId + protocolAddress?
```

Comment:
in the previous version the protocol address and chainId were used to identify the environment of the messages so maybe we should add them here if it's necessary?

Reply:
let's wait for if/when we have to manage several chains/protocol instances in the same dApp environment